templates/utils/auth0widget.js

```diff
@@ -84,7 +84,7 @@ export default dynamicSettings =>
                                             <div class="auth0-lock-form" id="content-container">
                                                 <div>
                                                 <p id="message">
-                                                    ${t('introduction')}
+                                                    ${t('introduction')} ${t('identities').replace('{{identities}}', identities)}.
```

Comment:
We may want this replace to be a regex where whitespace is allowed inside of `{{ identities}}`, for example, since someone could easily mistype it.